spec/dummy_sons_controller_spec.rb

```diff
@@ -71,89 +50,54 @@
                ]
              ] }]
         end
+        let(:pagination_params) do
+          { page: Wor::Paginate::Config.default_page, count: Wor::Paginate::Config.default_page,
+            total_count: model_count, total_pages: model_count, previous_page: 2, current_page: 3,
+            next_page: 4 }
+        end
 
         before do
           get :index_with_params
         end
 
-        it 'responds with page' do
-          expect(response_body(response)['page'].length).to be 1
-        end
+        include_examples 'proper pagination params'
 
         it 'responds with valid page' do
           expect(response_body(response)['page']).to eq expected_list
         end
+      end
 
-        it 'responds with count' do
-          expect(response_body(response)['count']).to be 1
+      context 'with a really high limit passed by option' do
+        let(:expected_list) do
+          dummy_models.first(50).map do |dummy|
```

Comment:
We really need first 50 ?

Reply:
:disappointed: my bad, you are only moving lines. We are going to refactor it in future :muscle: